backend/jest.config.js

```diff
@@ -9,7 +9,9 @@ module.exports = {
     '@telegram/(.*)': '<rootDir>/src/telegram/$1',
     '@test-utils/(.*)': '<rootDir>/src/test-utils/$1',
     '@shared/(.*)': '<rootDir>/../shared/src/$1',
+    '@mocks/(.*)': '<rootDir>/src/__mocks__/$1',
   },
+  modulePathIgnorePatterns: ['<rootDir>/build'],
```

Comment:
Not sure if I should do this or instead block typescript from building any mock files

Reply:
Sorry, could you explain more about what is it that you need to do here?

Reply:
Oops, my bad.

By default, my mock ts files (in `/src/__mock__`) get compiled into JS in `/build/__mock__`. When `jest` searches for mock modules to include, it will search both directories by default, resulting in a name collision.

The line above stops jest from looking inside `/build` for mock files to be included, but I'm not sure if i should instead add something to the `tsconfig` to stop it from building particular files (in this case mocks). The main downside of adding rules to `tsconfig` is that if I mock anything in the future in other folders (besides `/src/__mock__`), I'll need to manually exclude it too.